src/components/Info.js

```diff
@@ -8,8 +8,12 @@ export default function Info() {
             <p className="info--role"> Frontend Developer</p>
             <p className="info--website">nngrz</p>
             <div className="info--buttons">
-                <button className="btn--email"><i className="fa fa-envelope"></i> Email</button>
-                <button className="btn--linkedin"><i className="fa fa-linkedin-square" style={{size:"16px",color:"white"}}></i>LinkedIn</button>
+                <a href="mailto:ningrui1227@gmail.com" className="btn--email">
+                    <button className="btn--email"><i className="fa fa-envelope"></i> Email</button>
+                </a>
+                <a href="https://www.linkedin.com/in/ningrui-zhang1227" target="_blank" rel="noopener noreferrer">
+                    <button className="btn--linkedin"><i className="fa fa-linkedin-square" style={{size:"16px",color:"white"}}></i>LinkedIn</button>
```

Comment:
Please move <i className="fa fa-linkedin-square" style={{size:"16px",color:"white"}}></i> to a new lline. In general, the linke length should be <80 characters